.htaccess

```diff
@@ -61,20 +61,21 @@ RewriteRule "^downloads/releases(\/)?$" "/downloads" [R,END]
 # Cleanup various URLS with permanent redirects
 
 # /keyboards/{install|download|share}/{id}/ to /keyboards/x/id
-# RedirectMatch "^keyboards/(install|download|share)/([^/]+)/$" "/keyboards/$1/$2"
+# Remove final /
 RewriteRule "^keyboards/(install|download|share)/([^/]+)/$" "/keyboards/$1/$2" [R=301,END,QSA]
 
 # /keyboards/{id}/ to /keyboards/id
-RedirectMatch 301 "^keyboards/([^/]+)/$" "/keyboards/$1"
+# Remove final /
+RewriteRule "^keyboards/([^/]+)/$" "/keyboards/$1" [R=301,END,QSA]
 
-# /keyboards/ to /keyboards
-RedirectMatch 301 "^keyboards/$" "/keyboards"
+# /keyboards/ to /keyboards (Handled by other redirect)
+# Remove final /
 
 # Old share url /keyboards/{id}/share[/] to /keyboards/share/id
-RedirectMatch "^keyboards/(?!install|download|share)/share(/?)$" "/keyboards/share/$1"
+RewriteRule "^keyboards/(.*)/share(/?)$" "/keyboards/share.php?id=$1" [R=301]
```

Comment:
Instead of redirecting to a redirect, I short-circuit to

https://github.com/keymanapp/keyman.com/blob/adc4ede646cbad7bb866f1a36349016cfadc6b61/web.config#L120-L124

Reply:
Yeah, but the original is redirecting to a rewrite (not to a redirect), so it was:

1. `keyboards/khmer_angkor/share` redirects to `keyboards/share/khmer_angkor`
2. `keyboards/share/khmer_angkor` which rewrites to `/keyboards/share.php?id=khmer_angkor`

We don't want `/keyboards/share.php?id=khmer_angkor` to be visible to the visitor, just `/keyboards/share/khmer_angkor`. So I think we need to keep the original pattern?